cluster/addons/storage-crds/csinodeinfo.yaml

```diff
@@ -24,13 +24,17 @@ spec:
                   name:
                     description: The CSI driver that this object refers to.
                     type: string
+                    pattern: ^[a-zA-Z0-9][-a-zA-Z0-9_.]{0,61}[a-zA-Z0-9]$
+                    maxLength: 63
                   nodeID:
                     description: The node from the driver point of view.
                     type: string
                   topologyKeys:
                     description: List of keys supported by the driver.
                     items:
                       type: string
+                      pattern: ^([a-z0-9]([-a-z0-9.]{0,61}[a-z0-9])?[/])?[a-zA-Z0-9][-a-zA-Z0-9_.]{0,61}[a-zA-Z0-9]$
```

Comment:
first part is prefix, described in spec https://github.com/container-storage-interface/spec/blob/master/spec.md "// The key prefix MUST be 63 characters or less, begin and end with a"... second part after slash is same as above for driver name